### DISCUSSION TITLE
[feature request] HDR10+ metadata Generation

### DISCUSSION THREAD

**Discussion by blackops786187** [quietvoid/hdr10plus_tool#89] [Q&A]
Hi,

Is it possible to implement the ability to generate HDR10+ metadata and output it to a json. The only tool ive come across that can do this is davinci however since its a GUI based app, the process cannot be automated

**Comment by quietvoid** [comment]
See https://github.com/quietvoid/hdr10plus_tool/discussions/67#discussioncomment-9178965

The metadata is only profile A.

**Comment by rivertim** [comment]
@blackops786187 Resolve is capable of exporting profile B json files. This is offered via a drop-down in the export dialogue. Also I believe Resolve can be used with Python scripting language for automation.
